modules/ossm-control-plane-cli.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+[id="ossm-control-plane-deploy-cli_{context}"]
+= Deploying the control plane from the CLI
+
+Follow this procedure to deploy a basic `ServiceMeshControlPlane` from the command line.
+
+.Prerequisites
+
+* The {ProductName} Operator must be installed.
+* An account with the `cluster-admin` role.
+* Access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role.
++
+[source,terminal]
+----
+$ oc login https://{HOSTNAME}:6443
```

Comment:
This is outside of the scope of OSSM and really part of setting up oc so you can use your cluster. Should it be included here or assumed?

Reply:
I hear what you're saying @brian-avery . I suppose it's pretty obvious that you need to log in though. And if you aren't logged in, the terminal would tell you.

Reply:
We're going to leave it as is. It's for orienting our users. It's part of our ccs guidelines.